README.md

```diff
@@ -6,7 +6,7 @@ Monitor and analyze the emergent behaviors of Bitcoin networks.
 ## Major Features
 
 * Launch a bitcoin network with a specified number of nodes connected to each other according to a network topology from a graphml file.
-* Scenarios can be run across the network which can be programmed using the Bitcoin Core functional [test_framework language](https://github.com/bitcoin/bitcoin/tree/master/test/functional).
+* Scenarios can be run across the network which can be programmed using the Bitcoin Core functional [test_framework language](/test/functional).
```

Comment:
link doesn't work. you could direct to `src/test_framework` in warnet repo but there is no README there like the original link

Reply:
oops